gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -8,38 +8,91 @@ module ApplianceConsole
   class LogfileConfiguration
     LOGFILE_DIRECTORY = Pathname.new("/var/www/miq/vmdb/log").freeze
     LOGFILE_NAME = "miq_logs".freeze
+    MIQ_LOGS_CONF = Pathname.new("/etc/logrotate.d/miq_logs.conf").freeze
 
-    attr_accessor :disk
+    attr_accessor :size, :disk, :logrotate_count,
+                  :confirmed_new_disk, :requested_new_disk, :activated_new_disk,
+                  :confirmed_new_logrotate_count, :requested_new_logrotate_count, :activated_new_logrotate_count
 
     include ApplianceConsole::Logging
 
+    def initialize
+      self.requested_new_disk            = false
+      self.confirmed_new_disk            = false
+      self.activated_new_disk            = false
+      self.requested_new_logrotate_count = false
+      self.confirmed_new_logrotate_count = false
+      self.activated_new_logrotate_count = false
+
+      params = { "-h" => nil, "--output=size" => nil, nil => LOGFILE_DIRECTORY }
```

Comment:
Why are we doing human readable here? Wouldn't we always want the output in bytes?